http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -149,6 +142,13 @@ WORK_DIRECTORY='/mnt/disks/work'
 GSUTIL_CMD='docker run --rm -v /var:/var gcr.io/google-containers/toolbox:20200603-00 gsutil'
 GCLOUD_CMD='docker run --rm -v /var:/var gcr.io/google-containers/toolbox:20200603-00 gcloud'
 
+if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
+  export IS_RSTUDIO_RUNTIME="true"
+  export XDG_CONFIG_HOME=${RSTUDIO_USER_HOME}/.config/rstudio
```

Comment:
what XDG?

Reply:
oh, meant to remove this. it's for autosave which i'll use a separate PR for